### PR TITLE
Update Review Section Final

### PR DESCRIPTION
**Pull request by BryceSteuer**
(Apologies, I inadvertently pushed the first PR to webpage/upgrade rather then master)
Now we have a random review function present in the sidebar (for screens wide enough to support the sidebar).
This function appears in the "User Reviews" section for users with screens too small to view the sidebar.
These two renderings make the function load a bit more slowly, but it is to your discretion whether it is too slow.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -617,9 +576,20 @@ <h3><i class="fas fa-download"></i> Download</h3>
...
 
+  <script src="javascripts/testimony.js" type="text/javascript"></script>
+  <h4 id="reviewHeading">User Reviews</h4>
+   <div id="reviewFirst"></div>
+   <div id="reviewSecond"></div>
+   <div id="reviewThird"></div>
+   <div id="reviewFourth"></div>
+   <div id="reviewFifth"></div>
+
+
```
Comment (tuomas2) [inline index.html:587]: why are these two times? Also js file is loaded twice.

File: javascripts/testimony.js [hunk 2]
```diff
@@ -1,26 +1,59 @@
...
+		document.getElementById("reviewFourth").innerHTML = "<i>Excellent application, the best. You can see different versions, dictionaries in Greek and Hebrew (to English) and biblical commentaries ... it's like having the study library in your pocket. Thanks! <br/> <b>-A.B., AndBible user</b> <br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small>";
+	}
```
Comment (tuomas2) [inline javascripts/testimony.js:28]: Well, we can start with this, but I'd like to get this code abit more generic si tgat ut us easy to add more items to our review database.

I.e.

We have array of reviews:

```js
var reviews = [
{
 content: "content of review", // no need to have html there even.
 by: "name of reviewer or initials",
 stars: 5, // default, no need to specify to every item
 source: "Google Play",  // default, no need to specify to every item
},
...
];
```

and code would randomly pick 5 items from this list and render them nicely to the page.

File: javascripts/testimony.js [hunk 3]
```diff
@@ -1,26 +1,59 @@
...
+		document.getElementById("reviewFour").innerHTML = "<i>Excellent application, the best. You can see different versions, dictionaries in Greek and Hebrew (to English) and biblical commentaries ... it's like having the study library in your pocket. Thanks! <br/> <b>-A.B., AndBible user</b> <br/> <span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span><span class='fa fa-star checked'></span> <small>(Google Play)</small>";
+	}
+});
```
Comment (tuomas2) [inline javascripts/testimony.js:59]: Why are these twice?

File: javascripts/testimony.js (hunk 2 above)
Comment (tuomas2) [reply javascripts/testimony.js:28]: My idea is that we will go through our Google Play reviews and add *a lot* more of those here. Tens, even hundreds of 5-star reviews. This approach is not sustainable then any more.

**Comment by tuomas2** [comment]
> These two renderings make the function load a bit more slowly, but it is to your discretion whether it is too slow.

It feels fast enough now.

File: javascripts/testimony.js (hunk 2 above)
Comment (tuomas2) [reply javascripts/testimony.js:28]: Do you think this is doable to you? If not I could offer to help.

File: index.html (hunk 1 above)
Comment (BryceSteuer) [reply index.html:587]: The two renderings are for two different screen width contexts. The one in the sidebar only renders for browsers with a width above 900px I think. For lower widths, the sidebar is hidden. So in order for users with screen width less than 900px to still see the review function, I placed it again in a <div id="smScreen"> that only shows if the screen is less than 900px wide

Comment (tuomas2) [reply index.html:587]: I see, ok

File: javascripts/testimony.js [hunk 4]
```diff
@@ -1,26 +1,59 @@
-var slideIndex = 1;
-showSlides(slideIndex);
+/* Base code structure credit http://html-tuts.com/display-random-content-javascript/ */
```
Comment (tuomas2) [inline javascripts/testimony.js:1]: remove this file as it is not used any more

File: stylesheets/stylesheet.css [hunk 5]
```diff
@@ -1361,8 +1361,20 @@ q {
...
 }
+.checked {
+  color: orange;
+}
+.badge-success {
+  color: black;
+  background-color: #00cc00;
+  margin-bottom: 5px;
+}
+#reviewFirst, #reviewSecond, #reviewThird, #reviewFourth, #reviewFifth, #reviewHeading {
```
Comment (tuomas2) [inline stylesheets/stylesheet.css:1375]: not used, clean up

File: stylesheets/stylesheet.css [hunk 6]
```diff
@@ -1364,17 +1360,7 @@ q {
   border: 1px solid #cccccc;
   padding: 7.5px;
 }
-.checked {
-  color: orange;
```
Comment (tuomas2) [inline stylesheets/stylesheet.css]: stars are not any more yellow because of this

**Comment by tuomas2** [comment]
Thank you!

**Comment by BryceSteuer** [comment]
Thank _you_ for the privilege

**Pull request merged**